packages/compass-utils/package.json

```diff
@@ -71,5 +71,13 @@
     "prettier": "^2.7.1",
     "sinon": "^9.2.3",
     "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "@mongodb-js/compass-logging": "^1.1.7",
+    "glob": "^10.3.3"
+  },
+  "peerDependencies": {
+    "@mongodb-js/compass-logging": "^1.1.7",
+    "glob": "^10.3.3"
```

Comment:
compass-utils is really meant for small utilities that don’t have a place elsewhere in our package tree. I’d consider making this its own package.

Reply:
If i can vote .. I'd rather extract it and keep the logging centralised

Reply:
I ended up creating a new package for this.